graph-db/serverless/src/utils/cmr/fetchCollectionPermittedGroups.js

```diff
@@ -16,6 +33,9 @@ export const fetchCollectionPermittedGroups = async (conceptId, token) => {
 
   let response
   try {
+    // TODO: We can put a wait function here to ensure that this thing is waiting a long time between calls so that we can
```

Comment:
Remove, putting a wait before the call does not make sense, better to put waits between failures to the access-control app if there are any